### DISCUSSION TITLE
Since yesterday ruffle.js is not loading

### DISCUSSION THREAD

**Discussion by cestdani** [ruffle-rs/ruffle#6230] [Q&A]
https://brighteikaiwa.com/ge/index.html?load=MENU.SWF

This page was working fine yesterday, and then it just didn't. 
No changes were made to the code. 

`<script src="https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle" type="text/javascript">
`
returns:
`DevTools failed to load source map: Could not load content for https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

**Comment by torokati44** [comment]
The URL https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle works fine for me.
The linked page also appears to have a second `<script>` tag, with a fixed version number from 2022-02-06. Why is that there?
Also, have you tried cleaning your browser cache?
(Note that some jsDelivr caching issues are being worked on right now.)

**Comment by cestdani** [comment]
Thank you, I tried a few different ways to load ruffle. Fixed version number is gone now. Are you able to load/see https://brighteikaiwa.com/ge/index.html?load=MENU.SWF ?

The URL https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle works fine for me too.

Error:
DevTools failed to load source map: Could not load content for https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by danielhjacobs** [comment]
That's a console warning and doesn't matter. It's because of https://github.com/jsdelivr/jsdelivr/issues/18033. It's because the source map is at https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle/ruffle.js.map, not https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle.js.map.

**Comment by cestdani** [comment]
The console warning is because the main CDN is requesting the file, but it doesn't resolve.

**Comment by danielhjacobs** [comment]
Yes, which is because of the reason listed above. `<script src="https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle/ruffle.js"></script>` would let the request resolve, but the source map doesn't really matter for functionality.

**Comment by cestdani** [comment]
In any case, ruffle.js isn't loading

**Comment by danielhjacobs** [comment]
It's loading for me according to the network tab. The game is never loading however, since getQueryVariable() is never run, but if you open the console and manually run that function it loads.

**Reply by cestdani** [reply]
Thank you, it works now.
